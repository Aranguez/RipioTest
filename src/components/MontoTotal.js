import React from 'react';

const MontoTotal = props => {
    return (
        <div className="monto-total">
            <span>Billetera</span>
            <span className="t-azul">
                <i className="fa fa-bitcoin"></i> {props.totalAmount}
            </span>
        </div>
    );
}

export default MontoTotal;
