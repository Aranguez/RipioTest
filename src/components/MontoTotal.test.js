import React from 'react';
import { shallow } from 'enzyme';
import MontoTotal from './MontoTotal';

describe('montoTotal', () => {
    const montoTotal = shallow(<MontoTotal/>)
    
    it('renders properly ', () => {
        expect(montoTotal).toMatchSnapshot()
    });
})