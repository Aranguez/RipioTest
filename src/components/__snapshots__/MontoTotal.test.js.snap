// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`montoTotal renders properly  1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MontoTotal />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span>
          Monto actual
        </span>,
        <span
          className="monto t-azul"
        >
          <i
            className="fa fa-bitcoin"
          />
           
        </span>,
      ],
      "className": "monto-total",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Monto actual",
        },
        "ref": null,
        "rendered": "Monto actual",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <i
              className="fa fa-bitcoin"
            />,
            " ",
            undefined,
          ],
          "className": "monto t-azul",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fa fa-bitcoin",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          " ",
          undefined,
        ],
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span>
            Monto actual
          </span>,
          <span
            className="monto t-azul"
          >
            <i
              className="fa fa-bitcoin"
            />
             
          </span>,
        ],
        "className": "monto-total",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Monto actual",
          },
          "ref": null,
          "rendered": "Monto actual",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <i
                className="fa fa-bitcoin"
              />,
              " ",
              undefined,
            ],
            "className": "monto t-azul",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fa fa-bitcoin",
              },
              "ref": null,
              "rendered": null,
              "type": "i",
            },
            " ",
            undefined,
          ],
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
